server.js

```diff
@@ -1,23 +1,82 @@
 const express = require("express");
 const cors = require("cors");
+const uuid = require("uuid");
 
 const app = express();
+const port = process.env.PORT || 4500;
+
+// creating increment unique id for each message
+let generateNewID = 0;
 
 app.use(cors());
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+const welcomeMessage = [];
 
-const welcomeMessage = {
-  id: 0,
-  from: "Bart",
-  text: "Welcome to CYF chat system!",
-};
+// Time formating
+const today = new Date();
+const format = Intl.DateTimeFormat("en-us", {
```

Comment:
I would have prefered to use en-gb as the US time format has the month and day the oppisite way around so could be confusing